app/uk/gov/hmrc/vatsubscription/models/updateVatSubscription/RequestedChanges.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.vatsubscription.models.updateVatSubscription
+
+import play.api.libs.json.{Json, Writes}
+
+case class RequestedChanges(addressDetails: Boolean,
+                            returnPeriod: Boolean,
+                            repaymentBankDetails: Boolean)
+
+object RequestedChanges {
+
+  implicit val writes: Writes[RequestedChanges] = Writes {
+    model => Json.obj(
+      "PPOBDetails" -> model.addressDetails,
+      "returnPeriod" -> model.returnPeriod,
+      "repaymentBankDetails" -> model.repaymentBankDetails
```

Comment:
We will never submit changes for repayment BankDetails; so it is worth hard coding to false?